scss/6-components/forms/page-form.scss

```diff
@@ -0,0 +1,10 @@
+/** @define page-form; weak */
+.cads-page-form {
+  .cads-button {
+    margin-bottom: $cads-spacing-1;
+  }
+
+  &--single-button {
+    margin-top: $cads-spacing-4;
+  }
+}
```

Comment:
I really like the principle behind this, but rather than making this a page-level class I'd be tempted to treat this as a more localised button group. Perhaps `.cads-form-actions` to signify this is a group of buttons as part of a form? Does that match with what you are thinking of?

Reply:
I agree that a localised class would be better but not sure how to implement it here. I think the bit I was struggling with was that often if we have two buttons that are on the same page and they look like they should be part of the same group (e.g next/previous), they are functionally part of different forms. For example in decision trees the 'Next' button is part of the main question form, while the 'Previous' button is part of another form. Similar thing happens on the smart meter tool. I'm not sure how to address the spacing if I'm using a localised class in those cases. The only thing I can think of is trying out `first-of-type`.

Reply:
Ah I see that's interesting. I think first of type selectors would be fine here, as long as we can broadly wrap the `cads-form-actions` around the area where the buttons are?

Reply:
I've now tried with last of type instead. It didn't work in this case because of how the buttons are nested - i.e if I have 
something like:
scss:
```
.cads-form-actions {
  button:last-of-type {
    margin-bottom: $cads-spacing-1;
  }
}
```
html:
```
.cads-form-actions
  <form 1>
    <first button>
  <form 2>
    <second button>
```
it would think both buttons are last of type

Reply:
In which case I'd either go with what you had originally (treating it as a page-level class) or go even _more_ targeted and add a little utility class we can add to the specific buttons that we need. But I suspect the second option is less useful. So perhaps ignore me here!